src/config/default.json

```diff
@@ -2,7 +2,8 @@
 	"apiNodePort": 3001,
 	"endpoints": {
 		"marketData": "https://min-api.cryptocompare.com/",
-		"statisticsService": "https://symbol.services"
+		"statisticsService": "https://symbol.services",
```

Comment:
Should statistic service be removed?

Reply:
Yes, but I deleted in PR #1203 
because in this PR, the `statistic page` still using `statisticsService` endpoint